test/forms/FactCheckPageReviewFormProviderSpec.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import base.BaseSpec
+import models.YesNoAnswer
+import models.forms.FactCheckPageReview
+import play.api.data.{Form, FormError}
+
+class FactCheckPageReviewFormProviderSpec extends BaseSpec {
+
+  private trait Test {
+
+    val formProvider = new FactCheckPageReviewFormProvider()
+
+    val form: Form[FactCheckPageReview] = formProvider()
+
+    val answer = "answer"
+    val comment = "comment"
+
+    val definedComment = "This page is acceptable"
```

Comment:
We should really remove the comment related bits here. It is interesting that the tests still work!

Reply:
the non-impact comment related stuff intentionally left in. The form bind ignores what it doesnt want